### PR TITLE
Only allow one key per file in acl-add

### PR DESCRIPTION
**Pull request by callahad**
Otherwise, the additional keys get added without the sshcommand wrapper.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Can you add a testcase?

File: sshcommand [hunk 1]
```diff
@@ -136,6 +136,9 @@ sshcommand-acl-add() {
     KEY=$(cat "$KEY_FILE")
   fi
 
+  local count="$(wc -l <<< "$KEY")"
+  [ "$count" -eq 1 ] || log-fail "Too many keys provided, set one per invocation of sshcommand acl-add <USER> <NAME>"
```
Comment (michaelshobbs) [inline sshcommand]: Per [bash style guidelines](https://github.com/progrium/bashstyle), please use `[[` instead of `[`

**Comment by michaelshobbs** [comment]
I think you'll need to push this branch again to trigger a build. It was misconfigured initially.

File: sshcommand (hunk 1 above)
Comment (callahad) [inline sshcommand]: Fixed. Was just pulling that straight from dokku's [ssh-keys:add](https://github.com/dokku/dokku/blob/4e6a699140686a4700e821a18fd9c230d42f6ba9/plugins/ssh-keys/subcommands/add#L20-L21).

**Comment by callahad** [comment]
I can't get bats running correctly on my local machine, so I can't verify that the unit test is working, but it *should* be trivially correct...

**Comment by callahad** [comment]
Doh, `git commit -a` grabbed an unrelated change. Repushing...

**Comment by callahad** [comment]
There we go. CI is green

**Pull request merged**